web/html/src/components/picker/recurring-event-picker.tsx

```diff
@@ -360,7 +360,8 @@ class RecurringEventPicker extends React.Component<RecurringEventPickerProps, Re
                 </div>
                 <div className={`col-sm-1 ${styles.helpIcon}`}>
                   <i
-                    className="fa fa-question-circle spacewalk-help-link"
+                    className="fa fa-info-circle fa-1-5x"
+                    style={{ color: "royalblue" }}
```

Comment:
Not urgent, but would be nice to move inline styles into the theme instead, here and elsewhere too.